AGENTS.md

```
@@ -141,13 +181,11 @@ cargo run --release
 
 ### Feature Flags
 
-- `default`: `["json", "csv", "ssl", "additional_mysql_types", "verbose"]`
-- `ssl`: MySQL native TLS support using platform-native libraries (SChannel on Windows, SecureTransport on macOS, may use OpenSSL on Linux)
-- `ssl-rustls`: Pure Rust TLS implementation (alternative to native TLS)
+- `default`: `["json", "csv", "additional_mysql_types", "verbose"]`
 - `additional_mysql_types`: Support for BigDecimal, Decimal, Time, Frunk
 - `verbose`: Conditional logging via println!/eprintln!
 
-**Important**: `ssl` and `ssl-rustls` are mutually exclusive features.
+Note: TLS is now always available and is no longer a feature flag.
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)